QueryCiv3/Util.cs

```diff
@@ -1,11 +1,18 @@
 using System;
 using System.IO;
 using Blast;
+using System.Text.RegularExpressions;
+using System.Text;
 
 namespace QueryCiv3
 {
     public class Util
     {
+        // Encoding code page ID; 1252 is Civ3 encoding for US language version
+        //   See https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding?view=net-5.0 for other values
+        // NOTE: .NET 5 and later require a Nuget package and Encoder registration for these older encoding pages
+        static public Encoding Civ3Encoding = Encoding.GetEncoding(1252);
```

Comment:
Minor: Other code pages I know Civ uses are 1251 for Russian, and GBK for Chinese (which based on your link, the closest would be 936 in .NET).  The GBK I am only about 90% sure of; it matches screenshots I've seen of Chinese language Civ3 scenarios when I run it locally, but I don't know Chinese so I can't say with certainty that there isn't another, more complete encoding that's more appropriate.

For now, I'm thinking documenting known encodings used by Civ3 is sufficient; if the project takes off, we can use settings to swap out this variable as needed easily enough.

Reply:
Civ encoding is not something I'm familiar with at all. I didn't actually write those comments; I was just moving the string getting logic from somewhere else into a static method of the Util class so that I could call it from the new structs. So I'll defer to whoever actually wrote the above comments on what the most appropriate action is here, hehe.

Reply:
Oh, yup, looks like Puppeteer wrote that a month ago.  No need to modify it then, we'll do it if/when we get to that point.  Also no reason to be familiar with code pages nowadays unless you're working on a 20 year old game, or the internals of Windows with UTF-16.